recipes/PyAV/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "av" %}
```

Comment:
The name on pip and the actual Python package name is av. The project calls itself PyAV everywhere else. Which is preferred?